ExampleApp/Swift/ProductDetailViewController.swift

```diff
@@ -124,29 +124,30 @@ class ProductDetailViewController: OMSBaseViewController {
                   let textField = alertController.textFields?.first,
                   let text = textField.text,
                   let url = URL(string: text),
-                  let expectedReturnURL = URLComponents(string: "omiseExampleApp://omise_3ds_challenge"),
-                  let threeDSRequestorAppURL = expectedReturnURL.url?.absoluteString
+                  let deeplinkURL = URL(string: "omiseExampleApp://omise_3ds_challenge"),
+                  let expectedWebReturnURL = URL(string: "http://www.example.com/orders")
             else { return }
 
-            NetceteraThreeDSController.processAuthorizedURL(
-                url,
-                threeDSRequestorAppURL: threeDSRequestorAppURL,
-                uiCustomization: self.threeDSUICustomization,
-                in: self
-            )
-        })
-        present(alertController, animated: true, completion: nil)
-    }
+            AuthorizingPayment.shared.presentAuthPaymentController(
+                from: self,
+                url: url,
+                expectedWebViewReturnURL: expectedWebReturnURL,
+                deeplinkURL: deeplinkURL,
+                uiCustomization: self.threeDSUICustomization) { [weak self] result in
+                    print("AuthorizingPayment complete with result: \(result)")
+                    print("-")
```

Comment:
Do we need to log these or we forgot to remove?

Reply:
Example app normally print to console data comes from OmiseSDK, but as I added popup window with status I removed these lines with latest commit.